docs/content/en/functions/hmac.md

```diff
@@ -0,0 +1,35 @@
+---
+title: hmac
+# linktitle: hmac
+description: Hashes the given input and returns either an SHA1 or SHA256 checksum.
```

Comment:
This should be simplified to not include the list of hash functions.  Perhaps "Compute the cryptographic checksum of a message."